### DISCUSSION TITLE
LingmoOS Helium .. when it will be ready to download ?

### DISCUSSION THREAD

**Discussion by gfracasso** [LingmoOS/Community#29] [Bugs Report/缺陷提交] [answered]
Anybody know when Lingmo Helium will be ready to download ?
My best regards

**Comment by elysia-best** [comment, marked as answer]
> Anybody know when Lingmo Helium will be ready to download ? My best regards

I know how frustrating it can be when things don't go as planned, especially when we're facing a lack of human power and the urgent need for an update to Qt6. Rest assured, we're doing everything we can to get the 3.0 version out there as soon as we can. 

In the meantime, I've been working on a new image based on Debian12, and I'm really hopeful it'll be a stable version that works well for most of you.

Thank you so much for your patience and understanding during this time. We really appreciate your support and hope you'll continue to have a great day!

**Comment by gfracasso** [comment]
Hi Yinan Qin,
Thanks a lot for your update.
I really appreciate your work regarding the development of LingmoOS.
No problem to wait. I am shure that will be a great work.

Regarding your word “…in the meantime, I've been working on a new image
based on Debian12, and I'm really hopeful it'll be a stable version that
works well for most of you… “, please let me know when will be possible to
try its.

My best regards
Giovanni (from Italy)

Il giorno lun 23 dic 2024 alle ore 17:08 Yinan Qin ***@***.***>
ha scritto:
